.github/workflows/build.yml

```diff
@@ -48,46 +41,91 @@ jobs:
   Server:
     runs-on: ubuntu-latest
     env:
-      working-directory: ./server
-      INDEX_DB_CONFIG_USER: ${{ secrets.INDEX_DB_CONFIG_USER }}
```

Comment:
note: no longer need secrets for tests, config now stored in containers/test.env (message me for qa secrets)